src/Planner/PlannerJoinTree.cpp

```diff
@@ -40,6 +40,14 @@
 #include <Planner/PlannerActionsVisitor.h>
 #include <Planner/Utils.h>
 
+#include <AggregateFunctions/AggregateFunctionCount.h>
```

Comment:
Move headers thare are related to Processors at the top where we include headers related to Processors. These can be applied to all headers.